ev-rvs/nodejs-server-exercise
# feat(evobe-88): added jest test for pong endpoint and changed dockerfle.test to use --watchall for tdd

### Ticket
https://phunts.atlassian.net/browse/EVOBE-88

### Changes Made
- Added Jest test for `/pong` route

### Notes
*Format changes with updated VSCode settings (finally - my bad)*

### Steps to Test
1. Checkout this branch
2. Copy `env.example` as `.env` and update `.env` file with `postgres` initialization variables
3. Purge old docker volumes with `docker volumes prune` (optional)
4. Take down existing containers by running `docker-compose down`
5. Then run test containers by running `docker-compose -f test-docker-compose.yml up`
6. Verify test were successful - see screenshot
7. Stop test containers with `docker-compose down`
8. Restart the docker-compose group `docker-compose down -v && docker-compose up --build` optionally with `-d` as a daemon or without for running logs to container.
9. Go to `http:localhost:3003/pong` and you should see JSON output

<img width="697" alt="Screen Shot 2021-03-11 at 5 03 15 PM" src="https://user-images.githubusercontent.com/80280063/110861129-b2659980-828b-11eb-8002-07c290cf7ae5.png">
